data-access/Models/ScrollModels.cs

```diff
@@ -12,7 +12,7 @@ public class Edition
         public bool Locked { get; set; }
         public bool IsPublic {get; set;}
         public DateTime? LastEdit { get; set; }
-        public User Owner { get; set; }
+        public UserToken Owner { get; set; }
```

Comment:
Why are we returning the owner's token? I see no good use cases for this, just bad ones.